<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageTools</class>
 <widget class="qMRMLWidget" name="LanguageTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>554</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="8" column="0">
    <widget class="QPushButton" name="updateButton">
     <property name="toolTip">
      <string>Download, compile, and install latest translations</string>
     </property>
     <property name="text">
      <string>Update translation files</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="ctkCollapsibleButton" name="settingsCollapsibleButton">
     <property name="text">
      <string>Advanced settings</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="collapsedHeight">
      <number>6</number>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Qt 'lrelease' tool path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="lreleasePathLineEdit">
        <property name="toolTip">
         <string>Path to Qt lrelease executable. Leave empty to use default (the one bundled with the Slicer installer or available in system paths).</string>
        </property>
        <property name="settingKey">
         <string notr="true"/>
        </property>
        <property name="showHistoryButton">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="slicerVersionLabel">
        <property name="text">
         <string>Slicer version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="slicerVersionEdit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;master, 4.13, 4.14, ...&lt;/p&gt;&lt;p&gt;Only used for GitHub.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string notr="true">main</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="gitRepositoryLabel">
        <property name="text">
         <string>GitHub repository URL:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="githubRepositoryUrlEdit">
        <property name="toolTip">
         <string>Github repository that stores the translated strings.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="weblateDownloadUrlLabel">
        <property name="text">
         <string>Weblate download URL:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="weblateDownloadUrlEdit">
        <property name="toolTip">
         <string>Weblate project that stores the translated strings.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Translation file folders:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ctkFittedTextBrowser" name="translationFoldersTextBrowser">
        <property name="toolTip">
         <string>List of folders where the application looks for translations. This is for information only, the paths cannot be changed.</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="collapsed">
         <bool>true</bool>
        </property>
        <property name="showDetailsText">
         <string>Show more...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="languageSelectorLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Application language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkLanguageComboBox" name="languageSelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Change only takes effect after restarting the application.</string>
       </property>
       <property name="countryFlagsVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartButton">
       <property name="toolTip">
        <string>Restart the application to see updated translations on the graphical user interface.</string>
       </property>
       <property name="text">
        <string>Restart the application</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <widget class="QTextEdit" name="statusTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Input translations</string>
     </property>
     <property name="collapsedHeight">
      <number>6</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="localTsFolderLabel">
        <property name="text">
         <string>Input folder:</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="languagesLabel">
        <property name="text">
         <string>Languages:</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QRadioButton" name="weblateSourceRadioButton">
        <property name="text">
         <string>Weblate (latest translations)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QRadioButton" name="githubSourceRadioButton">
        <property name="text">
         <string>GitHub (updated daily)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QRadioButton" name="localTsFolderRadioButton">
        <property name="text">
         <string>Local folder (for offline use)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="latestTsFileOnlyLabel">
        <property name="text">
         <string>Latest file only:</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="languagesListRefreshButton">
        <property name="toolTip">
         <string>Refresh Weblate language list by querying the Weblate server.</string>
        </property>
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="ctkCheckableComboBox" name="languagesComboBox">
        <property name="toolTip">
         <string>Select language(s) that will be downloaded and installed.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="ctkPathLineEdit" name="localTsFolderPathLineEdit">
        <property name="toolTip">
         <string>Folder containing translation (.ts) files.</string>
        </property>
        <property name="settingKey">
         <string notr="true">Internationalization/LanguageTools/LocalTsFolderPathHistory</string>
        </property>
        <property name="nameFilters">
         <stringlist notr="true">
          <string>*</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="latestTsFileOnlyCheckBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If this option is checked then only the latest .ts file in the folder will be used. Enable this option to ignore previously downloaded .ts files are saved in the same folder.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Find text</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="collapsedHeight">
      <number>6</number>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Enable text finder:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="enableTextFindercheckBox">
          <property name="toolTip">
           <string>If enabled, then the shortcut opens the text finder, which can be used to extract text from the application user interface and open it in the translation website.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="textFinderShortcutLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Keyboard shortcut:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkCheckableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkFittedTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ctkFittedTextBrowser.h</header>
  </customwidget>
  <customwidget>
   <class>ctkLanguageComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkLanguageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>enableTextFindercheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableTextFindercheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>textFinderShortcutLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
